server/config/subject_page.proto

```diff
@@ -165,10 +165,10 @@ message ScatterTileSpec {
 
 message BarTileSpec {
   enum SortType {
-    ASCENDING = 1;
-    DESCENDING = 2;
-    ASCENDING_POPULATION = 3;
-    DESCENDING_POPULATION = 4;
+    ASCENDING = 0;
+    DESCENDING = 1;
+    ASCENDING_POPULATION = 2;
+    DESCENDING_POPULATION = 3;
```

Comment:
nit: Might make the default one as 0

Reply:
Following offline conversation of @keyurva, using `TYPE_UNSPECIFIED` as 0 to match proto conventions.